packages/sync/src/Functions.php

```diff
@@ -92,6 +124,14 @@ public static function sanitize_post_type( $post_type ) {
 		return (object) $sanitized_post_type;
 	}
 
+	/**
+	 * Return information about a synced post type.
+	 *
+	 * @param array  $sanitized_post_type Array of args used in constructing \WP_Post_Type.
+	 * @param string $post_type Post type name.
+	 *
+	 * @return object WP_Post_Type
```

Comment:
Could we prefix all global classes with `\` in docs too, since that's how they're referred to in a namespace?

Reply:
233c91be2